2-projects/1-todo-list/index.html

```diff
@@ -38,7 +38,7 @@
               </li>
               <li
                 class="list-group-item d-flex justify-content-between align-items-center"
-                style="text-decoration: line-through"
+                style="text-decoration: line-through;"
```

Comment:
You need to comment / remove all <li>s to avoid repeated <li> generated from the toDos array